README.md

```diff
@@ -26,12 +26,7 @@ Helpful hints
 
 ### Firefox on Android
 
-Firefox on Android uses the ````RC4-SHA```` SSL cipher suite.
-This must be enabled on your server in order for synchronization to work with Firefox on your smartphone or tablet.
-
-If you use *nginx* you need to add  ````RC4-SHA```` to the ```ssl_ciphers``` section in your ````nginx.conf````. For more information see the [nginx documentation](http://wiki.nginx.org/HttpSslModule#ssl_ciphers).
-
-If you use *apache* you need to add ```RC4-SHA``` to the ```SSLCipherSuite``` section in your ````apache2.conf````. For more information see the [apache2 documentation](https://httpd.apache.org/docs/2.2/mod/mod_ssl.html#sslciphersuite).
+Older versions of Firefox on Android used only the ````RC4-SHA```` SSL cipher suite. This has been fixed and "TLS_DHE_RSA_WITH_AES_256_CBC_SHA" and "TLS_DHE_RSA_WITH_AES_128_CBC_SHA" have been added to Firefox 29
```

Comment:
My first edit and I already do something wrong.. sorry I missed that 
full stop.
Do you know how I can correct this?

Oliver Gasser schreef op 2015-08-08 om 12:24:
>
> In README.md 
> <https://github.com/owncloud/mozilla_sync/pull/143#discussion_r36577095>:
>
> > @@ -26,12 +26,7 @@ Helpful hints
> >
> >  ### Firefox on Android
> >
> > -Firefox on Android uses the ````RC4-SHA```` SSL cipher suite.
> > -This must be enabled on your server in order for synchronization to work with Firefox on your smartphone or tablet.
> > -
> > -If you use *nginx* you need to add  ````RC4-SHA```` to the ```ssl_ciphers``` section in your ````nginx.conf````. For more information see the [nginx documentation](http://wiki.nginx.org/HttpSslModule#ssl_ciphers).
> > -
> > -If you use *apache* you need to add ```RC4-SHA``` to the ```SSLCipherSuite``` section in your ````apache2.conf````. For more information see the [apache2 documentation](https://httpd.apache.org/docs/2.2/mod/mod_ssl.html#sslciphersuite).
> > +Older versions of Firefox on Android used only the ````RC4-SHA```` SSL cipher suite. This has been fixed and "TLS_DHE_RSA_WITH_AES_256_CBC_SHA" and "TLS_DHE_RSA_WITH_AES_128_CBC_SHA" have been added to Firefox 29
>
> Full stop at the end of the paragraph is missing.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/owncloud/mozilla_sync/pull/143/files#r36577095>.
>

-- 
Met vriendelijke groet,
Sebastiaan Giebels